docs/resources/cdss_config_group.md

```diff
@@ -0,0 +1,23 @@
+# Resource: ncloud_cdss_config_group
+
+## Example Usage
+
+``` hcl
+resource "ncloud_cdss_config_group" "config-group" {
+  name = "from-tf-config"
+  kafka_version_code = "2823006"
+  description = "test"
+}
+```
+
```

Comment:
``` hcl
ncloud_cdss_config_group.config_group: Creating...
╷
│ Error: Plugin did not respond
│ 
│   with ncloud_cdss_config_group.config_group,
│   on main-cdss.tf line 24, in resource "ncloud_cdss_config_group" "config_group":
│   24: resource "ncloud_cdss_config_group" "config_group" {
│ 
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-ncloud plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x0 pc=0x101178c94]

goroutine 84 [running]:
github.com/terraform-providers/terraform-provider-ncloud/ncloud.resourceNcloudCDSSConfigGroupCreate({0x101564dc8, 0x1400007f4a0}, 0x0?, {0x1013c8ba0?, 0x1400007c940?})
        /Users/user/WORKS Drive/My Drive.localized/Coding/terraform/terraform-provider-ncloud/ncloud/resource_ncloud_cdss_config_group.go:60 +0x154
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0x1400045aa80, {0x101564e00, 0x1400038f5c0}, 0xd?, {0x1013c8ba0, 0x1400007c940})
        /Users/user/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.13.0/helper/schema/resource.go:708 +0xe8
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0x1400045aa80, {0x101564e00, 0x1400038f5c0}, 0x140003db6c0, 0x1400010bf00, {0x1013c8ba0, 0x1400007c940})
        /Users/user/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.13.0/helper/schema/resource.go:838 +0x714
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0x1400051ff38, {0x101564d58?, 0x1400025dd40?}, 0x140000afe00)
        /Users/user/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.13.0/helper/schema/grpc_provider.go:1021 +0xb70
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0x14000494140, {0x101564e00?, 0x1400038e5d0?}, 0x1400033acb0)
        /Users/user/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.8.0/tfprotov5/tf5server/server.go:812 +0x384
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0x1015229e0?, 0x14000494140}, {0x101564e00, 0x1400038e5d0}, 0x1400007ed80, 0x0)
        /Users/user/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.8.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:385 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0x1400023c700, {0x1015686f8, 0x14000512340}, 0x1400093a5a0, 0x14000283c20, 0x101aab9e0, 0x0)
        /Users/user/go/pkg/mod/google.golang.org/grpc@v1.45.0/server.go:1282 +0xb28
google.golang.org/grpc.(*Server).handleStream(0x1400023c700, {0x1015686f8, 0x14000512340}, 0x1400093a5a0, 0x0)
        /Users/user/go/pkg/mod/google.golang.org/grpc@v1.45.0/server.go:1619 +0x82c
google.golang.org/grpc.(*Server).serveStreams.func1.2()
        /Users/user/go/pkg/mod/google.golang.org/grpc@v1.45.0/server.go:921 +0x84
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /Users/user/go/pkg/mod/google.golang.org/grpc@v1.45.0/server.go:919 +0x290

Error: The terraform-provider-ncloud plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```


민간 리얼에서는 config group이 생성이 되지 않는데, 확인이 필요할거같습니다.

Reply:
@minosmlee 님, config group의 description값에 널체크 추가 했고 description값 변경에 따른 modify in place 동작도 추가하여 반영했습니다.

Reply:
감사합니다. 이제 잘 되네요!

Reply:
감사합니다. 이제 잘 되네요!